include/crm/common/ipc_attrd.h

```diff
@@ -46,6 +46,21 @@ typedef struct {
     } data;
 } pcmk__attrd_api_reply_t;
 
+/*!
+ * \internal
+ * \brief Get the value of an attribute from pacemaker-attrd
+ *
+ * \param[in] api           Connection to pacemaker-attrd
+ * \param[in] host          Look up the attribute on this host
+ *                          (or NULL for all hosts)
+ * \param[in] name          Attribute name
+ * \param[in] options       Bitmask of pcmk__node_attr_opts
+ *
+ * \return Standard Pacemaker return code
+ */
+int pcmk__attrd_api_query(pcmk_ipc_api_t *api, const char *host, const char *name,
+                          uint32_t options);
```

Comment:
Actually, this entire file could be made internal.  Perhaps it's best to just suffix the file with _internal and move on.

Reply:
Yep, that sounds good.

FYI the library-private headers allow for more efficient shared library linking since functions that are only used within the library can be marked as G_GNUC_INTERNAL. In this case we need to use the functions elsewhere so a regular internal header is better.